### ISSUE TITLE
loading remote context failed: http://schema.org/

### ISSUE THREAD

**Issue by darthmiguel** [jsonld-java/jsonld-java#180]
Hi,
I am trying to parse a simple json-ld example (taken from http://json-ld.org/playground/ tab Person)
{
  "@context": "http://schema.org/",
  "@type": "Person",
  "name": "Jane Doe",
  "jobTitle": "Professor",
  "telephone": "(425) 123-4567",
  "url": "http://www.janedoe.com"
}

I get the following exception:

```
Exception in thread "main" com.github.jsonldjava.core.JsonLdError: loading remote context failed: http://schema.org/
	at com.github.jsonldjava.core.DocumentLoader.loadDocument(DocumentLoader.java:31)
	at com.github.jsonldjava.core.Context.parse(Context.java:195)
	at com.github.jsonldjava.core.Context.parse(Context.java:142)
	at com.github.jsonldjava.core.Context.parse(Context.java:286)
	at com.github.jsonldjava.core.JsonLdApi.expand(JsonLdApi.java:543)
	at com.github.jsonldjava.core.JsonLdApi.expand(JsonLdApi.java:997)
	at com.github.jsonldjava.core.JsonLdProcessor.expand(JsonLdProcessor.java:146)
	at com.github.jsonldjava.core.JsonLdProcessor.toRDF(JsonLdProcessor.java:482)
	at com.github.jsonldjava.core.JsonLdProcessor.toRDF(JsonLdProcessor.java:534)
	at unibonn.eis.thesis.BigDataSemantification$.main(BigDataSemantification.scala:41)
	at unibonn.eis.thesis.BigDataSemantification.main(BigDataSemantification.scala)
```

I have tried with other examples and works fine.

**Comment by ansell** [comment]
Sorry, I am only seeing errors for schema.org and not any of the other hosts in the integration tests. I added some regression tests that are identical to others, but disabled them for now as it seems like server misconfiguration on their end.

You can try downloading the jsonld context manually, storing it on your classpath and referencing it using the instructions at:

https://github.com/jsonld-java/jsonld-java#loading-contexts-from-classpathjar

**Comment by ansell** [comment]
Hacked around this by using the Java native HttpURLConnection class to access schema.org instead of Apache HTTPClient. Ie, it seems to be an issue specific to Apache HTTPClient and schema.org. Since I haven't changed the version of Apache HTTPClient recently and the existing test that accessed schema.org (ContextCompactionTest) only started failing on Travis today, it would appear that Google broke it on their end with a recent update to schema.org.

Will be fixed in 0.8.4 when that is released.
